src/components/form/CreateModelForm.vue

```diff
@@ -160,16 +180,23 @@ async function onSubmit() {
 }
 
 /**
- * Set default model name on plugin name change.
+ * Set model path on plugin name change.
  */
 function onPluginChange() {
-  modelName.value = getPluginByName(modelPlugin.value).configuration.defaultFileName;
+  const { defaultFileName = '', isFolderTypeDiagram } = pluginConfiguration.value;
+
+  modelPath.value = isFolderTypeDiagram ? '' : defaultFileName;
 }
 
 onMounted(async () => {
   getAllModels(props.projectName).then((array) => {
     models.value = array;
   });
-  allFolders.value = await getProjectFolders(props.projectName);
 });
 </script>
+
+<style lang="scss">
```

Comment:
why is needed now but not before ?

Reply:
it's not needed, but better for UI
(removed `lang="scss"` and add `scoped`)

Reply:
ok but better keep `lang="scss"`

Reply:
ok